### PR TITLE
Fix delete floating IP for ACI driver

### PR DESCRIPTION
**Pull request by tbachman**
The delete floating IP precommit/postcommit calls
were passing the entire floating IP DB object. They
should only pass the floating IP ID.

Signed-off-by: Thomas Bachman <tbachman@yahoo.com>

### CODE REVIEW

File: networking_cisco/plugins/cisco/l3/drivers/asr1k/aci_asr1k_routertype_driver.py [hunk 1]
```diff
@@ -202,11 +202,11 @@ def update_floatingip_postcommit(self, context, fip_context):
 
     def delete_floatingip_precommit(self, context, fip_context):
         self.apic_driver.delete_floatingip_precommit(
-            context, fip_context.current)
+            context, fip_context.current['id'])
 
     def delete_floatingip_postcommit(self, context, fip_context):
         self.apic_driver.delete_floatingip_postcommit(
-            context, fip_context.current)
+            context, fip_context.current['id'])
```
Comment (kentwu1) [inline networking_cisco/plugins/cisco/l3/drivers/asr1k/aci_asr1k_routertype_driver.py:210]: Is there a UT for these 2 APIs?

Comment (tbachman) [reply networking_cisco/plugins/cisco/l3/drivers/asr1k/aci_asr1k_routertype_driver.py:210]: There is, but it doesn't cover these parameters. Will add.

**Comment by tbachman** [comment]
pushed new version, addressing reviewer comments.

File: networking_cisco/tests/unit/cisco/l3/test_aciasr1k_routertype_driver.py [hunk 2]
```diff
@@ -327,9 +332,52 @@ def _test_router_update_unset_gw_keeps_global_router(self,
...
+                r = router1['router']
+                self.l3_plugin._process_backlogged_routers()
+                # should now have one user-visible router, its single
+                # redundancy router and two global routers (one for each of
+                # the hosting devices of the aforementioned routers)
+                self._verify_ha_created_routers([r['id']])
+
+    def test_delete_floating_ip_pre_and_post(self):
+        #driver = mock.Mock()
+        #self.l3_plugin._get_router_type_driver = mock.Mock(
```
Comment (kentwu1) [inline networking_cisco/tests/unit/cisco/l3/test_aciasr1k_routertype_driver.py]: nit: can we just remove these commented-out lines?

Comment (tbachman) [reply networking_cisco/tests/unit/cisco/l3/test_aciasr1k_routertype_driver.py]: Will remove.

**Comment by tbachman** [comment]
Pushed new version, addressing reviewer comments.

**Pull request merged**